src/App.router.js

```diff
@@ -16,7 +18,18 @@ export const router = createBrowserRouter([
       },
       {
         path: '/',
-        element: <Search />
+        element: <Search />,
+        children: [
+          {
+            path: 'new',
+            element: <AddMovie/>
+          },
+          {
+            path: '/:movieId/edit',
```

Comment:
"Optionally, you can preserve current search query parameters, so that movie list is not updated when you click on the link."

Reply:
из заданий "Update your logic to select a movie. When a movie card is clicked, navigate the user to "/:movieId" route. We recommend to preserve current search params upon navigation, so that your movie list is not reset."